release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2010,6 +2010,18 @@ As a result, if you apply a performance profile, all nodes in the cluster reboot
 
+* An error might occur when deleting a pod that uses an SR-IOV network device. This error is caused by a change in {op-system-base} 9 where the previous name of a network interface is added to its alternative names list when it is renamed. As a consequence, when a pod attached to an SR-IOV virtual function (VF) is deleted, the VF returns to the pool with a new unexpected name, for example, `dev69`, instead of its original name, for example, `ensf0v2`. Although this error is non-fatal, Multus and SR-IOV logs might show the error while the system recovers on its own. Deleting the pod might take a few extra seconds due to this error. (link:https://issues.redhat.com/browse/OCPBUGS-11281[*OCPBUGS-11281*],link:https://issues.redhat.com/browse/OCPBUGS-18822[*OCPBUGS-18822*], link:https://issues.redhat.com/browse/RHEL-5988[*RHEL-5988*])
+
+* Starting from RHEL 5.14.0-284.28.1.el9_2, if you configure a SR-IOV virtual function with a specific MAC address, configuration errors might occur in the i40e driver. Consequently, Intel 7xx Series NICs might have connectivity issues. As a workaround, avoid specifying MAC addresses in the `metadata.annotations` field in the `Pod` resource. Instead, use the default address the driver assigns to the virtual function.(link:https://issues.redhat.com/browse/RHEL-7168[*RHEL-7168*],link:https://issues.redhat.com/browse/OCPBUGS-19536[*OCPBUGS-19536*],link:https://issues.redhat.com/browse/OCPBUGS-19407[*OCPBUGS-19407*], link:https://issues.redhat.com/browse/OCPBUGS-18873[*OCPBUGS-18873*])
+
+* Defining a `sysctl` value for a setting with slash in the name, for example for bond devices, in the `profile` field of a `Tuned` resource might not work. Values with a slash in the `sysctl` option name do not map correctly to the `/proc` filesystem. As a workaround, create a `MachineConfig` resource that places a configuration file with the required values to the `/etc/sysctl.d` node directory.(link:https://issues.redhat.com/browse/RHEL-3707[*RHEL-3707*])
+
+* Due to an issue with Kubernetes, the CPU Manager is unable to return CPU resources from the last pod admitted to a node to the pool of available CPU resources. These resources are allocatable if a subsequent pod is admitted to the node. However, this in turn becomes the last pod, and again, the CPU manager cannot return this pod's resources to the available pool. 
++
+This issue affects CPU load balancing features because these features depend on the CPU Manager releasing CPUs to the available pool. Consequently, non-guaranteed pods might run with a reduced number of CPUs. As a workaround, schedule a pod with a `best-effort` CPU Manager policy on the affected node. This pod will be the last admitted pod and this ensures the resources will be correctly released to the available pool.(link:https://issues.redhat.com/browse/OCPBUGS-17792[*OCPBUGS-17792*])
```

Comment:
```suggestion
* Currently, due to an issue with Kubernetes, the CPU Manager is unable to return CPU resources from the last pod admitted to a node to the pool of available CPU resources. These resources can be allocated when a subsequent pod is admitted to the node. However, this in turn becomes the last pod, and again the CPU manager cannot return the resources of this pod to the available pool.
+
This issue affects the CPU load balancing features because these features depend on the CPU Manager releasing CPUs to the available pool. Consequently, non-guaranteed pods can run with a reduced number of CPUs. As a workaround, schedule a pod with a best-effort CPU Manager policy on the affected node. This pod will be the last pod allowed, ensuring that resources are properly released to the available pool.(link:https://issues.redhat.com/browse/OCPBUGS-17792[*OCPBUGS-17792*])
```